src/main/kotlin/no/nav/familie/ks/sak/api/ForvaltningController.kt

```diff
@@ -66,6 +74,28 @@ class ForvaltningController(
         return ResponseEntity.ok(Ressurs.success(journalførDokumentResponse.journalpostId, "Dokument er Journalført"))
     }
 
+    @PostMapping("/opprett-oppgave")
+    fun opprettOppgave(@RequestBody opprettOppgaveDto: OpprettOppgaveDto): ResponseEntity<Ressurs<String>> {
+        // hent aktørId fra fnr
```

Comment:
Vi bør kalle tilgangService for å sjekke om saksbehandler har FORVALTER rolle til å kalle denne tjenesten.